frontend/store/collaterals.ts

```
@@ -33,13 +34,27 @@ export const mutations = {
 };
 
 export const actions = {
-    async fetchStepAndCut({ commit, dispatch, rootGetters }: ActionContext<State, State>) {
+    async fetchCoreValues({ commit, rootGetters }: ActionContext<State, State>) {
         const network = rootGetters['network/getMakerNetwork'];
         if (!network) {
             return;
         }
-        dispatch('setup');
         for (const collateral of Object.values(COLLATERALS)) {
+            const tokenAddress = await getTokenAddressByNetworkAndSymbol(network, collateral.symbol).catch(() => {
+                return undefined;
+            });
```

The same problem as before, please don't silently catch errors, display them